ouroboros-consensus-test/ouroboros-consensus-test.cabal

```diff
@@ -99,6 +99,7 @@ library
                      , quiet             >=0.2   && <0.3
                      , random
                      , serialise         >=0.2   && <0.3
+                     , singletons
```

Comment:
This commit (6f660ccc5e8944379897df866fec8f4810ff7c7b) as well (squashed)

Reply:
I prefer smaller commits which updates a single component at a time, which allows to focus more on smaller chunks related to each other.